commands/create_command.py

```diff
@@ -0,0 +1,41 @@
+from Icommand import ICommand
+import os, json, sys
+parent_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), os.pardir))
+sys.path.append(parent_dir)
+from SchemaKeys import SchemaKeys
+
+class CreateCommand(ICommand):
+    def __init__(self, schema_path):
+        self.__validate_and_load_schema(schema_path)
+        self.execute()
+
+    def __validate_and_load_schema(self, schema_path):
+        if (schema_path== None or not os.path.exists(os.path.join(parent_dir, schema_path))):
+            raise Exception("FileNotFound")
```

Comment:
Are you going to throw exception or status code ?
Exception is fine, but in case of success what the command will return ?
It should return an output has a result.
Because it doesn’t need a status any more because you replaced tha status with an exception

Result will be an object of json says database succeeded in case of create.
In get, it will return the object.